src/routes/orders.js

```diff
@@ -1,95 +1,90 @@
 const express = require("express");
+const bodyParser = require("body-parser");
+let jsonParser = bodyParser.json();
 let router = express.Router();
 
-const fs = require("fs");
-const path = require("path");
-const shortid = require("shortid");
-
-let productsDataBase = require("../db/all-products.json");
-let usersDataBase = require("../db/all-users.json");
-
-const getProducts = productsOrder => {
-  const productsOrderIds = productsOrder.products;
-  const getProductsResult = [];
-
-  productsOrderIds.map(item => {
-    const getOneProduct = productsDataBase.find(
-      element => element.id === Number(item)
-    );
-    if (getOneProduct) {
-      getProductsResult.push(getOneProduct);
-    }
-  });
-
-  return getProductsResult;
+const mongoose = require("mongoose");
+const Order = require("../../schemas/orders");
+
+const checkOrder = orderToCheck => {
+  const creator = orderToCheck.creator;
+  const productsList = orderToCheck.productsList;
+  const deliveryType = orderToCheck.deliveryType;
+  const deliveryAdress = orderToCheck.deliveryAdress;
+  const sumToPay = orderToCheck.sumToPay;
+  if (
+    typeof creator === "string" &&
+    typeof productsList === "object" &&
+    typeof deliveryType === "string" &&
+    typeof deliveryAdress === "string" &&
+    typeof sumToPay === "number"
+  )
+    return true;
+  else return false;
 };
 
-const makeOrderDir = order => {
-  let getUserById;
-  usersDataBase.map(user => {
-    if (user.id === order.user) {
-      getUserById = user;
-    }
-  });
-
-  const userDirPath = path.join(
-    __dirname,
-    "../",
-    "db/",
-    "users",
-    `/${getUserById.user}`,
-    "/orders"
-  );
-
-  fs.mkdir(userDirPath, err => {
-    if (err) {
-      return console.log(err);
-    }
-  });
-
-  let orderName = [];
-  order.products.map(item => {
-    let productSearch = productsDataBase.find(elem => elem.id === Number(item));
-
-    if (productSearch) {
-      orderName.push(productSearch.name);
-    }
-  });
-
-  const orderDirPath = path.join(`${userDirPath}`, `${orderName}.json`);
-
-  fs.writeFile(orderDirPath, JSON.stringify(order), err => {
-    if (err) {
-      return console.log(err);
-    }
-    console.log(`New order file "${orderName}.json" already created!`);
-  });
-};
-
-router.post("/*", (req, res) => {
-  let body = "";
-
-  req.on("data", data => {
-    body = body + data;
-    console.log("Incoming order data!");
-  });
+router.get("/:orderId", (req, res) => {
+  const id = req.params.orderId;
+  Order.findById(id)
```

Comment:
Also I would like to see 404 error in case order would not be found. Rather than 200 response with `undefined` order field